graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/processors/PipelineInterpreter.java

```diff
@@ -368,7 +368,7 @@ private void evaluateStage(Stage stage,
         interpreterListener.exitStage(stage);
     }
 
-    private boolean executeRuleActions(Rule rule,
+    public boolean executeRuleActions(Rule rule,
```

Comment:
I think this method can remain `private`.

Reply:
fixed